tools/publicapigen/main.go

```diff
@@ -32,10 +32,11 @@ type registeredConstant struct {
 }
 
 var (
```

Comment:
Note to self, we should put a `README.md` in this directory explaining what this tool is, and how to use it (both from a "execute" view point, but also a using directives in the runtime viewpoint)